mix.exs

```diff
@@ -45,7 +44,7 @@ defmodule Avalanche.MixProject do
       {:jason, "~> 1.3"},
       {:joken, "~> 2.6"},
       {:nimble_options, "~> 0.4 or ~> 1.0"},
-      {:mentat, "~> 0.7.1"},
```

Comment:
we should unlock deps though, right? There was no change in `mix.lock` other than additions. I want some red 😂

Reply:
Turns out I had a few minutes!